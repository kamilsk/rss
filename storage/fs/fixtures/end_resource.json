{
  "id": "end_resource",
  "name": "end resource",
  "src": "https://rss.octolab.net/kamilsk/podcasts"
}
