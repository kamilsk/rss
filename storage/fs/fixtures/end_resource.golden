{"id":"end_resource","name":"end resource","src":null}
